src/mixins/TransformMixin.vue

```diff
@@ -0,0 +1,23 @@
+<script>
+import marked from 'marked';
+
+export default {
+  name:    'TransformMixin',
+  methods: {
+    /**
+     * Parse text to markdown format.
+     *
+     * @param {string} text - Text to parse.
```

Comment:
In method body you have check on existing `text` variable, but here says that it can be only string. Maybe should change `{string|null}` ?

Reply:
Yes, that's right. I updated this.